README.md

```diff
@@ -10,14 +10,19 @@ Service APIs implemented:
 - Amazon SimpleDB
 - Amazon Mechanical Turk
 - Amazon CloudWatch (MON)
-- Amazon DynamoDB (ddb2) 
+- Amazon DynamoDB (ddb2)
 
-All API functions have been implemented.  Not all functions have been thoroughly tested, so exercise care when integrating this library into production code.  Please send bug reports and patches.
+All API functions have been implemented.  Not all functions have been
```

Comment:
Actually, a single line break is harmless to markdown. See [here](https://daringfireball.net/projects/markdown/syntax#p):

    A paragraph is simply one or more consecutive lines of text, separated by
    one or more blank lines. (A blank line is any line that looks like a blank
    line — a line containing nothing but spaces or tabs is considered blank.)
    Normal paragraphs should not be indented with spaces or tabs.

    The implication of the “one or more consecutive lines of text” rule is that
    Markdown supports “hard-wrapped” text paragraphs. This differs significantly
    from most other text-to-HTML formatters (including Movable Type’s “Convert
    Line Breaks” option) which translate every line break character in a
    paragraph into a <br /> tag.

Some people are bothered by this, but it's not broken.